config/locales/en/views/layouts.yml

```diff
@@ -3,9 +3,15 @@ en:
   layouts:
     application:
       cookies:
-        accept_cookies: Accept cookies
-        cookie_banner_message_html: We use <a class="govuk-link" href="%{page_path}">cookies to collect information</a> about how you use this service.
-        refuse_cookies: Refuse cookies
+        accept_cookies: Accept analytics cookies
+        analytics_cookies: We’d also like to use analytics cookies so we can understand how you use the service and make improvements.
+        confirmation_accept: You've accepted additional cookies.
+        confirmation_link_to_settings: You can <a class="govuk-link" href="%{href}">change your cookie settings</a> at any time.
```

Comment:
I would recommend appending `_html` to this key to mark it as hmtl_safe

Reply:
Done